integration/test_sync_apis.js

```diff
@@ -26,7 +26,11 @@ module.exports = {
         ackWorkerInterval: 100, 
         schedulerInterval: 100, 
         schedulerLockName: 'test:syncApi:lock', 
-        useCache: true});
+        useCache: true,
+        cuidGenerator: function(params) {
```

Comment:
This may be little bit confusing around naming. The proper way to explain this is that clients should generate ClientID. What we can do on the server is to give a chance how cuid is being picked (for example by not using __fh header. I really like this change but I think that we should change it from 
`generator` to `selector` as we do not want people to think that we generating client id's on the server :D